rero_ils/modules/documents/utils.py

```diff
@@ -499,3 +503,18 @@ def create_contributions(contributions):
 
         calculated_contributions.append(contribution)
     return calculated_contributions
+
+
+def get_remote_cover(isbn):
+    """Document cover service."""
+    cover_service = current_app.config.get('RERO_ILS_THUMBNAIL_SERVICE_URL')
+    url = cover_service + '?height=60px&jsonpCallbackParam=callback'\
+                          '&type=isbn&width=60px&callback=thumb&value=' + isbn
+    response = requests.get(
+        url, headers={'referer': flask_request.host_url})
+
+    if response.status_code != 200:
+        current_app.logger.warning(
```

Comment:
It will log a looooooooot of message isn't it ? Maybe `warning` level isn't the best for this message ? info/debug ?